src/WebJobs.Script/runtimeassemblies.json

```diff
@@ -34,10 +34,6 @@
       "name": "Microsoft.Identity.Client.Extensions.Msal",
       "resolutionPolicy": "private"
     },
```

Comment:
What new dependencies are being brought in? Some of those might already be present, but this file should be updated with any new dependencies to make sure the host is not sharing them

Reply:
Within the WebHost `Azure.Identity` and `Azure.Security.KeyVault.Secrets` were added. The identity dependency is already in this file (I notice that it is already imported in `WebJobs.Script.csproj` , so the only change would be to add the `Azure.Security.KeyVault.Secrets` dependency here?

Does it matter that these were added to `WebJobs.Script.WebHost` and not `WebJobs.Script`?

Reply:
No, ultimately, the Web Host will have all the dependencies, direct or transitive.

What you're describing aligns with what I expected. `Azure.Identity` was brought in with the secretless work. `Azure.Security.KeyVault.Secrets` should be added.